config/txgh.yml

```diff
@@ -1,15 +0,0 @@
-txgh:
-    github:
-        repos:
-            matthewjackowski/txgh-test-resources:
-                api_username: <%= ENV['GITHUB_USERNAME']  %>
-                api_token: <%= ENV['GITHUB_TOKEN'] %>
-                push_source_to: <%= ENV['GITHUB_PUSH_SOURCE_TO'] %>
-                branch: <%= ENV['GITHUB_BRANCH'] %>
-    transifex:
-        projects:
-            txgh-test-1:
-                tx_config: <%= ENV['TX_CONFIG_PATH'] %>
-                api_username: <%= ENV['TX_USERNAME'] %>
-                api_password: <%= ENV['TX_PASSWORD'] %>
-                push_translations_to: <%= ENV['TX_PUSH_TRANSLATIONS_TO'] %>
```

Comment:
That's a good question. The lumos version I'm working on writes this file before deploying. If you're using Heroku however, I understand that this needs to remain in the repo. I'll put a modified version back in.